### PR TITLE
FUSETOOLS-2689: UI Tests - Try to manipulate with routes via Jolokia

### PR DESCRIPTION
**Pull request by djelinek**
Signed-off-by: Dominik Jelinek <djelinek@redhat.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaManipulationTest.java [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+		connectionWizard.setTextJolokiaURL(conf.getUrl());
+		if (!conf.isVerifySSL())
+			connectionWizard.toggleDoNOTVerifySSLCertificatesCHB(true);
+		connectionWizard.finish();
+	}
+
+	/**
+	 * Opens JMX Navigator view and delete Jolokia connection
+	 */
+	public void deleteConnection() {
```
Comment (tsedmik) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaManipulationTest.java]: IMHO this method could be private

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaManipulationTest.java [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+		activateRoute();
+
+		// Activate editor and set breakpoint
+		CamelEditor editor = new CamelEditor("<connected>Remote CamelContext: SampleCamel");
+		editor.activate();
+		editor.setBreakpoint("Transform transform1");
+
+		WaitCondition rememberCondition = new ShellIsAvailable("Confirm Perspective Switch");
+		new WaitUntil(rememberCondition, TimePeriod.MEDIUM);
```
Comment (tsedmik) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaManipulationTest.java]: What about if you are in _Debug_ perspective? This condition will throw an exception. I believe you wanted to use: `new WaitUntil(rememberCondition, TimePeriod.MEDIUM, false);`

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaManipulationTest.java [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+	 * <li>Click resume (route should have again stops on breakpoint)</li>
+	 * <li>Remove breakpoint</li>
+	 * <li>Click resume (route should have continue)</li>
+	 * <li>Verify Error log does not contains any <i>Fuse</i> errors</li>
+	 * </ul>
+	 */
+	@Test
+	public void testDebuggingRoute() {
+
+		activateRoute();
```
Comment (tsedmik) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/JolokiaManipulationTest.java]: On the first sight, it was a bit confusing for me. What about changing the name of the method to something more descriptive like _activateRouteEditing_

**Comment by djelinek** [comment]
ready for review

**Pull request merged**